src/foam/graphics/CView.js

```diff
@@ -1305,14 +1299,21 @@ foam.CLASS({
   ],
 
   methods: [
+    async function init() {
+      let el = await this.el();
+      this.context = el.getContext('2d');
+      this.context3D = el.getContext('webgl');
```

Comment:
We don't want to grab these if we don't need them. We'll only use one, so they should still be factories.
Maybe just switch the factories to use el_().

Reply:
addressed